src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,160 @@
+                + "─────||──███████████──||─────\n"
+                + "──────||─███████████─||──────\n"
+                + "───────|||||||||||||||───────\n"
+                + "─────────████||█████─────────\n"
+                + "────────█████||██████────────\n"
+                + "──────███████||███████───────\n"
+                + "─────█████──█||█──█████──────\n"
+                + "─────█████──████──█████──────\n"
+                + "──────████──████──████───────\n"
+                + "──────████──████──████───────\n"
+                + "──────████───██───████───────\n"
+                + "──────████───██───████───────\n"
+                + "──────████──████──████───────\n"
+                + "─██────██───████───██─────██─\n"
+                + "─██───████──████──████────██─\n"
+                + "─███████████████████████████─\n"
+                + "─██─────────████──────────██─\n"
+                + "─██─────────████──────────██─\n"
+                + "────────────████─────────────\n"
+                + "─────────────██──────────────\n";
+
+
+
+        System.out.println(LOGO);
+        Scanner input = new Scanner(System.in);
+        System.out.println("Hello! I'm Jigsaw\n");
+        System.out.println("What can I do for you?\n");
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+        boolean isActive = true;
+        while (isActive) {
+            String command = input.nextLine();
+            String[] commands = command.split(" ",2);
+            try {
+                switch (commands[0]) {
+                case "bye":
+                    System.out.println("Bye. Hope to see you again soon!\n");
+                    isActive = false;
+                    break;
+                case "list":
+                    for (int i = 0; i < taskCount; i++) {
+                        System.out.println((i + 1) + "." + tasks[i].toString());
+                    }
+                    break;
+                case "mark":
+                    tasks[Integer.parseInt(commands[1]) - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println(tasks[Integer.parseInt(commands[1]) - 1].printTask());
+                    break;
+                case "unmark":
+                    tasks[Integer.parseInt(commands[1]) - 1].markAsUndone();
+                    System.out.println("OK, I've marked this task as not done yet:");
+                    System.out.println(tasks[Integer.parseInt(commands[1]) - 1].printTask());
+                    break;
+                case "todo":
+                    Todo todo = new Todo(commands[1]);
+                    tasks[taskCount] = todo;
+                    taskCount ++;
+                    System.out.println("Got it I have added this task:");
+                    System.out.println("  " + todo.toString());
+                    System.out.println("Now you have " + taskCount + " tasks in the list.");
```

Comment:
Instead of using  "System.out.println()" thrice, you can use it only once.
Example:
 System.out.println( "Got it I have added this task:" +"\n"
                    +"  " + todo.toString() +"\n" +
                    + "Now you have " + taskCount + " tasks in the list."+"\n");